input/templates/my-account-edit-address.json

```diff
@@ -2,13 +2,19 @@
 	"header" : { "$ref" : "common/header.json" },
 	"footer" : { "$ref" : "common/footer.json" },
 	"content" : {
-		"shippingAddress" : {
-			"title" : "Shipping Address",
-			"name" : "Bruce Wayne",
-			"address" : "100 Gotham Street",
-			"region" : "Gotham",
-			"postalCode" : "11111",
-			"country" : "Country"
+		"addressForm" : {
+			"id" : "34jhd94",
+			"salutations" : { "$ref" : "form/choose-title-1.json" },
+		  "firstName" : "Bruce",
+		  "lastName" : "Wayne",
+		  "streetName" : "100 Gotham Street",
+		  "additionalStreetInfo" : "Basement",
+		  "city" : "Gotham",
+		  "region" : "NY",
+		  "postalCode" : "11111",
+      "countries" : { "$ref" : "form/choose-country-1.json" },
+      "defaultShippingAddress" : true,
+      "defaultBillingAddress" : false
```

Comment:
This info is not coming directly from our platform, but from the form (which might receive the info from the platform or from the user input). It has many different fields and more complex stuff, that's why in this case we cannot reuse the `address-1.json` from before.